hyperparameter_hunter/compat/keras_optimization_helper.py

```diff
@@ -56,7 +56,9 @@
 except ModuleNotFoundError:
     base_keras_callback = type("PlaceholderBaseKerasCallback", (), {})
     BaseKerasInitializer = type("PlaceholderBaseKerasInitializer", (), {})
-
+# when working on cuda10.1, no information will be output, but pass and stop
+except Except as e:
+    print(e)
 
```

Comment:
Can you add an empty line here to keep the original two lines between the exception catching and the function definition below?